src/hu_stuff.c

```diff
@@ -1476,8 +1476,8 @@ void WI_DrawTimeWidget(void)
 void HU_Erase(void)
 {
   // [FG] TODO optimize!
-  if (!automapactive && viewwindowx)
-    R_DrawViewBorder();
+  // if (!automapactive && viewwindowx)
+  //   R_DrawViewBorder();
```

Comment:
@fabiangreffrath Why do we need this? Seems to work without it.

Reply:
The boom HUD draws on the border and this makes sure to clean it up.

Reply:
Oh, there were `HUlib_erase` functions. I guess it's not worth optimizing.